src/building/how-to-build-and-run.md

```diff
@@ -30,23 +30,56 @@ settings (and possibly others, such as `llvm.ccache`):
 
 ```toml
 [llvm]
-# Enables LLVM assertions, which will check that the LLVM bitcode generated
-# by the compiler is internally consistent. These are particularly helpful
-# if you edit `codegen`.
+# Indicates whether the LLVM assertions are enabled or not
 assertions = true
 
 [rust]
-# This will make your build more parallel; it costs a bit of runtime
-# performance perhaps (less inlining) but it's worth it.
```

Comment:
Was this from `config.toml.example`? It looks like these two lines explained why the `= 0` value was set, and this was lost.

Reply:
Yes, this is from `config.toml.example`. I'll add it back in https://github.com/rust-lang/rust/pull/76141.